docs/fides/docs/getting_started/quickstart.md

```diff
@@ -0,0 +1,177 @@
+  - For Mac, see [Homebrew's `make` formulae](https://formulae.brew.sh/formula/make)
+  - For Windows, see this [`make` installation guide](http://gnuwin32.sourceforge.net/packages/make.htm)
+
+2. Install a version of `docker` that includes support for `docker-compose`. We recommend installing [Docker Desktop](https://www.docker.com/products/docker-desktop) for Mac and Windows. For more information about Docker Compose, see "[Overview of Docker Compose](https://docs.docker.com/compose/)" on the Docker docs site.
+
+## Step 1: Clone the [Fidesctl repo](https://github.com/ethyca/fides)
+
+1. Clone the [Fidesctl repo](https://github.com/ethyca/fides).
+
+```bash
+$ git clone git@github.com:ethyca/fides.git                 
+> Cloning into 'fides'...
+> remote: Enumerating objects: 4855, done.
+> remote: Counting objects: 100% (2485/2485), done.
+> remote: Compressing objects: 100% (1095/1095), done.
+> remote: Total 4855 (delta 1460), reused 1696 (delta 1071), pack-reused 2370
+> Receiving objects: 100% (4855/4855), 14.66 MiB | 2.81 MiB/s, done.
+> Resolving deltas: 100% (2620/2620), done.
+```
+
+2. Move into the `fides` directory, which contains the Makefile we want to work with.
+```
+$ cd fides
+```
+
+## Step 2: Start the `fidesctl` container to run `fidesctl` CLI
+
+To build the interactive shell where you can run `fidesctl` CLI commands, run `make cli` from your fides directory. The first time you run `make cli` in your Fides repository, it may take around 3 minutes to build. 
+
+You're ready to run `fidesctl` CLI commands once you see the `fidesctl#` prompt.
+
+```bash
+$ make cli
+Build the images required in the docker-compose file...
+...
+Building fidesapi
+...
+Building fidesctl
+...
+Building docs
+...
+root@1a742083cedf:/fides/fidesctl#
+```
+
+## Step 3: Initialize `fidesctl` CLI
+
+Now you're ready to build the required images, spin up the database, and run other initialization scripts with the `fidesctl init-db` command.
+
+```bash
+$ ~/fides/fidesctl# fidesctl init-db
+> INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
+> INFO  [alembic.runtime.migration] Will assume transactional DDL.
+```
+
+## Step 4: Confirm your `fidesctl` CLI can reach the server
+
+To confirm that `fidesctl` can reach the server and everything is ready to go, run `fidesctl ping`.
+
+```bash
+root@796cfde906f1:/fides/fidesctl# fidesctl ping
+Pinging http://fidesctl:8080/health...
+{
+  "data": {
+    "message": "Fides service is healthy!"
+  }
+}
+```
+
+## Step 5: Review the example files.
+
+The fides repository you cloned includes a set of example files in the `fides/fidesctl/demo_resources` directory written in YAML and using the [Fides configuration syntax](/language/syntax/) and [Fides Taxonomy](/tutorial/taxonomy/).
+
+These files represent your organization's privacy polcy, relevant data, registry, and system configuration. 
+
+- `demo_dataset.yml` 
```

Comment:
Ideally, this list could include a description of each file at a high level and how they might relate. (Or a diagram can show their relationships and include an explanation underneath it.)